src/script/widgets/ClassificationPanel.js

```diff
@@ -0,0 +1,317 @@
+/**
+ * Copyright (c) 2008-2011 The Open Planning Project
+ *
+ * Published under the GPL license.
+ * See https://github.com/opengeo/gxp/raw/master/license.txt for the full text
+ * of the license.
+ */
+
+Ext.namespace("gxp");
+
+gxp.ClassificationPanel = Ext.extend(Ext.Panel, {
+
+    hidden: false,
+
+    rulePanel: null,
+
+    classifyText: "Classify",
```

Comment:
please also add least add this to the locale files (https://github.com/boundlessgeo/gxp/tree/master/src/script/locale), for the other languages just copy in the English language.